client.ts

```diff
@@ -175,12 +177,40 @@ export interface ClientBatchCheckResponse {
   result: ClientBatchCheckSingleResponse[];
 }
 
+export const OnDuplicateWrite = WriteRequestWritesOnDuplicate;
+export const OnMissingDelete = WriteRequestDeletesOnMissing;
+
+export type OnDuplicateWrite = WriteRequestWritesOnDuplicate;
+export type OnMissingDelete = WriteRequestDeletesOnMissing;
+
+export interface ClientWriteConflictOptions {
+  onDuplicateWrite?: OnDuplicateWrite;
+  onMissingDelete?: OnMissingDelete;
```

Comment:
Thanks for the PR @phamhieu!

To maintain consistency with the API and other SDKs, would you mind pluralizing these?

So:

```suggestion
  onDuplicateWrites?: OnDuplicateWrites;
  onMissingDeletes?: OnMissingDeletes;
```

Reply:
Oh no! Apologies for this miscommunication there @phamhieu - will chat with @daniel-jonathan.

But having multiple folks reviewing, we sometimes miss or contradict each other - sorry!

The aim is to have the SDKs more or less consistent with each other unless there's a strong reason not to.